jekyll/_config.yml

```diff
@@ -21,10 +21,14 @@ plugins:
   - jekyll-target-blank
   - jekyll-toc
   - jekyll-last-modified-at
+  - jekyll-timeago
 
 last-modified-at:
   date-format: '%d-%b-%y'
 
+jekyll_timeago:
+  depth: 1
```

Comment:
Can you please add a quick comment here to explain why `depth: 1`. I think that would be useful for future people.